components/ChipTextField/examples/UITextFieldWithChipsExample.swift

```diff
@@ -0,0 +1,265 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Should we even include this example? I think we've made more progress on the MDCTextField subclass and this experiment doesn't need to be included anymore